pkg/kyverno/common/common.go

```diff
@@ -918,11 +920,13 @@ func processMutateEngineResponse(policy *v1.ClusterPolicy, mutateResponse *respo
 		if mutateLogPath == "" {
 			mutatedResource := string(yamlEncodedResource) + string("\n---")
 			if len(strings.TrimSpace(mutatedResource)) > 0 {
-				if !stdin {
+				if !stdin && printPatchResource {
 					fmt.Printf("\nmutate policy %s applied to %s:", policy.Name, resPath)
 				}
-				fmt.Printf("\n" + mutatedResource)
-				fmt.Printf("\n")
+				if printPatchResource {
+					fmt.Printf("\n" + mutatedResource)
+					fmt.Printf("\n")
```

Comment:
Can this be done in single line only ? Any reason why you have done in two different lines ? Also Why do we need to give a newline at the starting and ending of mutatedResource ?

```
fmt.Printf("\n" + mutatedResource + "\n") 
```

Reply:
> 
> 
> Can this be done in single line only ? Any reason why you have done in two different lines ? Also Why do we need to give a newline at the starting and ending of mutatedResource ?
> 
> ```
> fmt.Printf("\n" + mutatedResource + "\n") 
> ```
yes, only one "\n" is required.